masterlist.yaml

```diff
@@ -5973,6 +5973,12 @@ plugins:
       - crc: 0xA4E835CA
         util: 'SSEEdit v4.0.3'
 
+  - name: 'Cathedral - 3D Pine Grass.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42032/' ]
+    after:
+      - 'Cathedral Landscapes.esp'
+      - 'Cathedral - 3D Mountain Flowers.esp'
```

Comment:
`Cathedral - 3D Mountain Flowers.esp` is a master, so `Cathedral - 3D Pine Grass.esp` will always load after. Just in case, is there another version of `Cathedral - 3D Mountain Flowers.esp` other than the one hosted [here](https://www.nexusmods.com/skyrimspecialedition/mods/41312/) that isn't a master?

Reply:
Right, it is esm-flagged. I removed it in the update.